src/javascript/javascript-parser.ts

```diff
@@ -27,6 +27,16 @@ declare class SyntaxError {
   column: number;
 }
 
+export interface Comment {
```

Comment:
Please document these types and why they're declared.

Also, if `type` is one of a small set of strings, specify it that way, something like:

```typescript
interface Comment {
  type: 'single-line' | 'multi-line';
}
```